### PR TITLE
unit-tests-jest

### PR DESCRIPTION
**Pull request by kevin-l1**
image

### CODE REVIEW

File: unit-tests-jest/numbers.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ * It works for both integer and fractional numbers.
+ * That is, `toDollars(12)` returns `$12.00` and
+ * `toDollars(11.341)` returns `$11.34`.
+ */
+export function toDollars(amount) {
+  return `$${Number.parseFloat(amount).toFixed(2)}`;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.js:15]: `.parseFloat` is not needed. You can just use `amount.toFixed(2)`

**Review by RobertGardner — APPROVED** [review]
More edge case tests would be nice.
